docs/spec.md

```diff
@@ -778,28 +778,28 @@ defined as follows:
 
 Afterwards, within each `process_group`:
 
-* `result@process[result_index] = exec(schedule)` for some binary tree
-  `schedule` where:
+* `result@process[result_index] = convert_or_quantize(exec(schedule), type(result))`
+  for some binary tree `schedule` where:
   * `exec(node)` = `computation(exec(node.left), exec(node.right))`.
   * `exec(leaf)` = `leaf.value`.
 * `schedule` is an implementation-defined binary tree whose in-order
-  traversal is `operands@process_group...[result_index]`.
+  traversal is `convert_or_quantize(operands@process_group...[result_index], type(func_inputs(computation)[0]))`.
 
 #### Inputs
 
 | Label | Name                    | Type                                                             | Constraints |
 |-------|-------------------------|------------------------------------------------------------------|-------------|
-| (I1)  | `operand`               | tensor                                                           | (C5), (C6)  |
+| (I1)  | `operand`               | tensor or quantized tensor                                       | (C5), (C6)  |
```

Comment:
Thanks for the catch! It was unintended and now fixed.

Reply:
>> The reduction will be more complex if different scale is used for the left and right elements being applied to the computation function

I agree that this will not be straight-forward. The individual operations within the computation blocks are supposed to handle per-axis computation. However, we do not have per-axis support for the entire opset.